lib/moog-require.js

```diff
@@ -62,7 +63,12 @@ module.exports = function(options) {
         self._indexes = glob.sync(self.options.localModules + '/**/index.js');
```

Comment:
We could exclude all npm namespace folder from the list:

```suggestion
        self._indexes = glob.sync(`${self.options.localModules}/**/index.js`, {
          ignore: `${self.options.localModules}/@*/**`
        });
```

Reply:
The legitimate issue could also occur with namespaced modules, they could appear under two parent dirs at project level using this feature.